src/sagas/RxjsSaga.js

```diff
@@ -14,13 +14,16 @@ export default class RxjsSaga {
    *
    * @param {Function} Saga implementation
    * @param {any} initial model
+   * @param {Function} dispatch function
```

Comment:
We need to do the same for `RxjsSaga` even though this bug is not present while using rxjs saga. But we still need to keep SagaAbstraction interface.